Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs

```diff
@@ -175,21 +185,42 @@ protected static IdentityDbContext IdentityDbContext
             }
         }
 
-        public HttpClient HttpClient { get; }
+        private readonly TaskCompletionSource<HttpClient> _httpClientTaskCompletionSource;
+
+        public Task<HttpClient> HttpClient => _httpClientTaskCompletionSource.Task;
 
         private TestServer CreateIdentityTestServer(string storageProvider)

```

Comment:
With a couple test servers getting created, what if we consolidate the code into a method and create the TestServer with this method and only the custom settings for each test server should be different.